app/backplane/radio_module/src/c_radio_module.cpp

```diff
@@ -25,7 +25,7 @@ void CRadioModule::AddTenantsToTasks() {
     // Networking
     networkingTask.AddTenant(sensorModuleListenerTenant);
     networkingTask.AddTenant(powerModuleListenerTenant);
-    networkingTask.AddTenant(tftpServerTenant);
```

Comment:
Intentional. Maybe we bring it back, but given that we listen for a lot of other stuff, I'm nervous about keeping TFTP in here compared to other modules